src/renderer/components/TokenSale.vue

```diff
@@ -216,6 +217,7 @@ export default {
           this.$services.alerts.success(
             this.$t('tokenSaleSuccessful', { symbol: res.symbol, balance: res.balance }),
           );
+          // TODO: we should add to user assets right away here and then add onlyFetchUserAssets: true to fetch call
           this.$store.dispatch('fetchHoldings', { done: null });
```

Comment:
Yeah I knew the done: null could be removed everywhere; I left it everywhere here and on mobile; since it makes it readily apparent that it could take a done callback, but I think it’s fine to remove it everywhere; may do that later.